wagn/features/update_includers.feature

```diff
@@ -8,6 +9,7 @@ Feature: Updates for Children of watched cards
   Background:
     Given I am signed in as Joe User
     And I create Book card "Ulysses"
+    And Jobs are dispatched
```

Comment:
I'm super happy this is being tested.

I will make the pedantic comment that this would fail the "Vishal should understand it" test for whether the wording makes sense to show on a high level.  But I'm not sure how best to do this, unless we are actually going to translate waiting for jobs to finish into a specific user experience.